### ISSUE TITLE
upgrade to whisper.cpp 1.5.5

### ISSUE THREAD

**Issue by jwijffels** [bnosac/audio.whisper#64]
https://github.com/ggerganov/whisper.cpp/releases/tag/v1.5.5

**Comment by jmgirard** [comment]
Any chance we could update to 1.6.2?

**Comment by jwijffels** [comment]
Yes, I think I will make that effort somewhere in September. Which changes in whisper.cpp are important for you that are newer than 1.5.4

**Comment by jmgirard** [comment]
Mostly I want to try the Flash Attention added in 1.6.0

**Comment by jwijffels** [comment]
Yes, that's also the main thing I think is relevant from my end as well.

**Comment by jmgirard** [comment]
@jwijffels Any chance you know how to compile ffmpeg with CUDA support? Been trying to follow [this guide](https://docs.nvidia.com/video-technologies/video-codec-sdk/12.2/ffmpeg-with-nvidia-gpu/index.html) but it always fails.

**Comment by jwijffels** [comment]
> @jwijffels Any chance you know how to compile ffmpeg with CUDA support? Been trying to follow [this guide](https://docs.nvidia.com/video-technologies/video-codec-sdk/12.2/ffmpeg-with-nvidia-gpu/index.html) but it always fails.

I've only worked with the av bindings and not with compiling ffmpeg with CUDA support myself. I see you have https://github.com/jmgirard/tidymedia which interfaces with a ffmpeg binary. I haven't looked into compiling ffmpeg myself.

**Comment by jmgirard** [comment]
It was a hassle but I finally got it working.

**Comment by jwijffels** [comment]
Closing as absolete due to upgrade https://github.com/bnosac/audio.whisper/releases/tag/0.5.0
